quickfixj-core/pom.xml

```diff
@@ -1,490 +1,368 @@
+			<scope>runtime</scope>
+			<optional>true</optional>
+		</dependency>
+		<dependency>
+			<groupId>com.sleepycat</groupId>
+			<artifactId>je</artifactId>
+			<version>18.3.12</version>
+			<optional>true</optional>
+		</dependency>
+	</dependencies>
 
-        <dependency>
-            <groupId>org.apache.mina</groupId>
-            <artifactId>mina-core</artifactId>
-            <version>2.1.6</version>
-        </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-            <version>${slf4j.version}</version>
-        </dependency>
+	<build>
+		<resources>
+		</resources>
 
-        <dependency>
-            <groupId>com.cloudhopper.proxool</groupId>
-            <artifactId>proxool</artifactId>
-            <version>0.9.1</version>
-            <optional>true</optional>
-            <exclusions>
-                <!-- bug in proxool pom - should be test scope -->
-                <exclusion>
-                    <groupId>avalon-framework</groupId>
-                    <artifactId>avalon-framework-api</artifactId>
-                </exclusion>
-                <!-- we use slf4j which provides this API -->
-                <exclusion>
-                    <groupId>commons-logging</groupId>
-                    <artifactId>commons-logging</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <groupId>com.cloudhopper.proxool</groupId>
-            <artifactId>proxool-cglib</artifactId>
-            <version>0.9.1</version>
-            <optional>true</optional>
-            <exclusions>
-                <!-- bug in proxool pom - should be test scope -->
-                <exclusion>
-                    <groupId>avalon-framework</groupId>
-                    <artifactId>avalon-framework-api</artifactId>
-                </exclusion>
-                <!-- we use slf4j which provides this API -->
-                <exclusion>
-                    <groupId>commons-logging</groupId>
-                    <artifactId>commons-logging</artifactId>
-                </exclusion>
-            </exclusions>
-        </dependency>
-        <dependency>
-            <!-- slf4j facade for JCL which is required by proxool -->
-            <groupId>org.slf4j</groupId>
-            <artifactId>jcl-over-slf4j</artifactId>
-            <version>${slf4j.version}</version>
-            <scope>runtime</scope>
-            <optional>true</optional>
-        </dependency>
-        <dependency>
-            <groupId>com.sleepycat</groupId>
-            <artifactId>je</artifactId>
-            <version>18.3.12</version>
-            <optional>true</optional>
-        </dependency>
-    </dependencies>
+		<testResources>
+			<testResource>
+				<directory>src/test/resources</directory>
+			</testResource>
+			<testResource>
+				<directory>src/main/resources</directory>
+			</testResource>
+		</testResources>
 
-    <build>
```

Comment:
@david-gibbs-ig do you recall why you removed this? Formerly we had the XML dictionaries packaged into the core JAR, which does not work without these lines. Not sure right now if we really need the dictionaries in the core JAR, though.

Reply:
@chrjohn  Yes, that's right. In this design the dictionaries should not be in core now. The will only come from (message) jars that have been added to the classpath. These can be the default ones provided by QFJ or custom jars.